internal/rbd/rbd_attach.go

```diff
@@ -397,6 +399,30 @@ func waitForrbdImage(ctx context.Context, backoff wait.Backoff, volOptions *rbdV
 	return err
 }
 
+// statergicActionOnLogFile act on log file based on cephLogStrategy.
+func statergicActionOnLogFile(ctx context.Context, logStrategy, logFile string) {
```

Comment:
@pkalever may be you meant strategicAction.. ?

Reply:
right!